### PR TITLE
Weather in schedule: change units to imperial

### PR DESCRIPTION
**Pull request by dmitrymarchuk**
Follows up https://github.com/sparkletown/sparkle/pull/2092
@sofisparkle requested a change to imperial units

### CODE REVIEW

File: src/api/openweathermap.ts [hunk 1]
```diff
@@ -15,9 +15,9 @@ type OpenWeatherApiResponse = {
 };
 
 export type WeatherInfo = {
-  temperatureCelsius: number;
```
Comment (goran-peoski-work) [inline src/api/openweathermap.ts:18]: do not remove, may be useful in the future, put ?

File: src/api/openweathermap.ts [hunk 2]
```diff
@@ -15,9 +15,9 @@ type OpenWeatherApiResponse = {
...
+  temperatureFahrenheit: number;
   stateOfSky: string;
-  windSpeedKPH: number;
```
Comment (goran-peoski-work) [inline src/api/openweathermap.ts:20]: do not remove, may be useful in the future, put ?

File: src/components/molecules/NavBarScheduleWeather/NavBarScheduleWeather.tsx [hunk 3]
```diff
@@ -41,15 +41,15 @@ export const NavBarScheduleWeather: React.FC<ContainerClassName> = ({
           )}
         >
           <div className="NavBarScheduleWeather__temp-and-state">
-            <h3>{`${weather.temperatureCelsius} °C`}</h3>
```
Comment (goran-peoski-work) [inline src/components/molecules/NavBarScheduleWeather/NavBarScheduleWeather.tsx:44]: do not remove, may be useful in the future,
put `const USE_IMPERIAL = true;`  just under the imports
with a @debt comment to extract it into util later on
use that flag to switch these two.

File: src/components/molecules/NavBarScheduleWeather/NavBarScheduleWeather.tsx [hunk 4]
```diff
@@ -41,15 +41,15 @@ export const NavBarScheduleWeather: React.FC<ContainerClassName> = ({
...
+            <h3>{`${weather.temperatureFahrenheit} °F`}</h3>
             <h5 className="NavBarScheduleWeather__state">
               {weather.stateOfSky}
             </h5>
           </div>
           <div>
             Humidity <b>{weather.humidityPercents}</b>% — Wind{" "}
             <b>
-              {weather.windSpeedKPH} KPH {weather.windDirection}
```
Comment (goran-peoski-work) [inline src/components/molecules/NavBarScheduleWeather/NavBarScheduleWeather.tsx:52]: do not remove, may be useful in the future,
put `const USE_IMPERIAL = true;`  just under the imports
with a @debt comment to extract it into util later on
use that flag to switch these two.

File: functions/openweathermap.js [hunk 5]
```diff
@@ -5,7 +5,7 @@ const { parseJson } = require("./src/utils/misc");
 
 const fetchWeather = async () => {
   const url =
-    "https://api.openweathermap.org/data/2.5/weather?lat=40.787&lon=-119.206&units=metric&lang=en&" +
```
Comment (goran-peoski-work) [inline functions/openweathermap.js:8]: do not remove, may be useful in the future,
put `const USE_IMPERIAL = true;`  just under the imports
with a @debt comment to extract it into util later on
use that flag to switch these two.

**Review by goran-peoski-work — CHANGES_REQUESTED** [review]
Do not remove older code, use a flag to switch for now, refactor later.

**Comment by mike-lvov** [comment]
@goran-peoski-work given that this code is only for burn-staging/burn(and is not going to be ported to the main platform in the nearest future), it's ok to delete the code.

But overall approach with flags is smart & I think we should start introducing it in our main `staging`

File: src/api/openweathermap.ts (hunk 1 above)
Comment (mike-lvov) [reply src/api/openweathermap.ts:18]: https://github.com/sparkletown/sparkle/pull/2173#issuecomment-908932019

File: src/api/openweathermap.ts (hunk 2 above)
Comment (mike-lvov) [reply src/api/openweathermap.ts:20]: https://github.com/sparkletown/sparkle/pull/2173#issuecomment-908932019

File: src/components/molecules/NavBarScheduleWeather/NavBarScheduleWeather.tsx (hunk 3 above)
Comment (mike-lvov) [reply src/components/molecules/NavBarScheduleWeather/NavBarScheduleWeather.tsx:44]: https://github.com/sparkletown/sparkle/pull/2173#issuecomment-908932019

File: src/components/molecules/NavBarScheduleWeather/NavBarScheduleWeather.tsx (hunk 4 above)
Comment (mike-lvov) [reply src/components/molecules/NavBarScheduleWeather/NavBarScheduleWeather.tsx:52]: https://github.com/sparkletown/sparkle/pull/2173#issuecomment-908932019

File: functions/openweathermap.js (hunk 5 above)
Comment (mike-lvov) [reply functions/openweathermap.js:8]: https://github.com/sparkletown/sparkle/pull/2173#issuecomment-908932019

**Review by mike-lvov — APPROVED** [review]
Given the comment above, approving this to introduce to the main platform

**Comment by goran-peoski-work** [comment]
> @goran-peoski-work given that this code is only for burn-staging/burn(and is not going to be ported to the main platform in the nearest future), it's ok to delete the code.

The act of deleting the code: 1 minute.
The act of adding a flag in the code: 2 minute.
The act of reinventing the wheel in the future: X > 2 minutes.

**Pull request merged**